### PR TITLE
monitoring: explain how to compose DSN string

### PR DESCRIPTION
**Pull request by moio**
Feel free to copy edit!

Context from this change comes from internal mailing list:

http://mailman.suse.de/mlarch/SuSE/suse-manager/2019/suse-manager.2019.12/msg00173.html

### CODE REVIEW

File: modules/administration/pages/monitoring.adoc [hunk 1]
```diff
@@ -217,7 +217,7 @@ image::monitoring_enable_services.png[scaledwidth=80%]
 
 . In the {susemgr} {webui}, open the details page of the system to be monitored, and navigate to the menu:Formulas[Prometheus Exporters] tab.
 . Check the [guimenu]``Enabled`` checkbox for both the Node and the Postgres Exporter.
-. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the path to your data source (for example, [systemitem]``postgresql://user:passwd@localhost:5432/database?sslmode=disable``).
+. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the the string [systemitem]``postgresql://USER:PASSWORD@localhost:5432/database?sslmode=disable``, replacing ``USER`` and ``PASSWORD`` with the values of the ``db_user`` and ``db_password`` from the ``/etc/rhn/rhn.conf`` file.
```
Comment (keichwa) [inline modules/administration/pages/monitoring.adoc]:
```suggestion
. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the string [systemitem]``postgresql://USER:PASSWORD@localhost:5432/database?sslmode=disable``, replacing ``USER`` and ``PASSWORD`` with the values of ``db_user`` and ``db_password`` from the ``/etc/rhn/rhn.conf`` file. 
```

**Comment by Loquacity** [comment]
This has conflicted with @cavalheiro 's Monitoring edits from PR#20 https://github.com/uyuni-project/uyuni-docs/pull/20 Easy enough to resolve technically, but I'm not sure what the correct information is.

**Comment by cavalheiro** [comment]
The updated version of the monitoring admin guide does not include specific arguments for each exporter, as those are explained in detail in the linked upstream documentation. If we want to include PostgreSQL exporter specifics, it would make sense to include it for others as well. Is this needed?

**Comment by Loquacity** [comment]
OK, in that case, I recommend dropping this PR.

File: modules/administration/pages/monitoring.adoc (hunk 1 above)
Comment (moio) [reply modules/administration/pages/monitoring.adoc]: I agree we should always use PostgreSQL, which is the correct name

**Pull request closed without merging**